tools/simple_hydrog/share/make_simple_hydrog.csh

```diff
@@ -31,19 +31,11 @@
 # =========================================================================
 
 if (`gfdl_platform` == "hpcs-csc") then
-    wipetmp
-else
-    if ($?TMPDIR) then
-        echo WARNING
-        echo "WARNING: Intended for PP/AN use; may not work properly on this site"
-        echo WARNING
-        echo
-    else
-        echo 'FATAL: $TMPDIR needed'
-        exit 1
-    endif
+    echo "WARNING: Intended for PP/AN use; may not work properly on this site"
 endif
 
+set my_tempdir=`mktemp -d`
```

Comment:
Suggest checking the exit status in case user can't write to TMPDIR:

```
if ( $status ) then
    echo ERROR: Cannot create temporary directory
    exit 1
else

Reply:
OK. Will add.